crates/diesel_models/src/payment_attempt.rs

```diff
@@ -303,9 +290,10 @@ impl PaymentAttemptNew {
     /// returns amount + surcharge_amount + tax_amount (surcharge) + shipping_cost + order_tax_amount
     pub fn calculate_net_amount(&self) -> MinorUnit {
```

Comment:
Why this function is required when we have `get_total_amount` from `NetAmount`

Reply:
This `impl` is on diesel models of PaymentAttemptNew, in which net_amount is still a MinorUnit